src/status_im/contacts/events.cljs

```diff
@@ -264,26 +253,31 @@
   :load-default-contacts!
   [(inject-cofx ::get-default-contacts-and-groups)]
   (fn [{:keys [db default-contacts default-groups]} _]
-    (let [{:keys [groups] :contacts/keys [contacts]} db]
+    (let [{:contacts/keys [contacts] :group/keys [contact-groups]} db]
       {:dispatch-n (concat
-                     (prepare-default-groups-events groups default-groups)
+                     (prepare-default-groups-events contact-groups default-groups)
                      (prepare-default-contacts-events contacts default-contacts)
                      (prepare-add-chat-events contacts default-contacts)
                      (prepare-bot-commands-events contacts default-contacts)
                      (prepare-add-contacts-to-groups-events contacts default-contacts))})))
```

Comment:
It's not necessary to bother with removing nil values from `:dispatch-n` vector, as reagent already handles this: https://github.com/Day8/re-frame/blob/master/src/re_frame/fx.cljc#L130